lisa/tools/kill.py

```diff
@@ -15,12 +20,12 @@ def command(self) -> str:
     def can_install(self) -> bool:
         return False
 
-    def by_name(self, process_name: str, signum: int = 9) -> None:
+    def by_name(self, process_name: str, signum: int = SIGKILL) -> None:
```

Comment:
This may not be right in Windows, but we cannot do that more so far.

Reply:
Yep, it will need a windows version once we need it there. Can I resolve this?